src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/help-modelCoverageArtifact.html

```diff
@@ -1,6 +1,9 @@
 <div>
-<br>Generate a Cobertura model coverage report, <b>coberturamodelcoverage.xml</b>, and save it in the <b>matlabTestArtifacts</b> folder of the Jenkins workspace.
-This report includes coverage results for Simulink&#174; models that are tested using Simulink Test&#8482;.<br>
-<br>Note: This feature requires a <b>Simulink Coverage&#8482;</b> license and is supported only in MATLAB R2018b or a newer release.<br>
+<br>
+Generate a model coverage report in Cobertura XML format. By default, MATLAB names the artifact <b>coberturamodelcoverage.xml</b> and stores it in the <b>matlabTestArtifacts</b> folder of the project workspace. 

```

Comment:
I would reorganize the text here (moving the third sentence in between the first and second).

Generate a model coverage report in Cobertura XML format. This artifact includes coverage results for Simulink&#174; models that are tested using Simulink Test&#8482;. By default, MATLAB names the artifact <b>coberturamodelcoverage.xml</b> and stores it in the <b>matlabTestArtifacts</b> folder of the project workspace.

Reply:
Done.